ansible/roles/edu-sharing/tasks/edu-sharing-docker/backup-alfresco-data.yml

```diff
@@ -15,16 +15,3 @@
     PGPASSWORD="{{alfresco_db.password}}" pg_dump -h localhost -U {{alfresco_db.user}}  {{alfresco_db.name}}  --serializable-deferrable | gzip > {{backup_path}}/{{alfresco_db.name}}-dump.sql.gz
   tags:
     - backup-alfresco-data
-
-- name: Create backup directory for alfresco data.
-  archive:
-    path: "{{ alf_home}}/alf_data/*"
-    dest: "{{ backup_path }}/alfresco-backup.tar.gz"
-    format: gz
-  tags:
-    - migration-alfresco-data
-
-- name: Ensure tomcat is started.
-  shell: "{{tomcat_start_command}}"
-  tags:
-    - backup-alfresco-data
```

Comment:
Confusing to stop the tomcat in this file and start it in another file. Both files (backup-alfresco-data.yml and migrate-alfresco-data.yml) will just work if processed together. Maybe make just one file out of it?

Btw: why is the tomcat started? It is the webserver for the old edu-sharing version, isn't it? After updating the version 7, the old version should not run, but be stopped by the Ansible scripts, right?

Reply:
Hello @mirjan-hoffmann 

yes, you are right, but this pull is only for fixing migration and I don't want to add other changes to it.

Btw: this is something that I will do when we want to make it ready for PROD, since now we are still developing and testing.

In end, I will give a last hand, but since every change we are going to test in our server, I think is OK to merge it.